content/2-4.md

```diff
@@ -193,21 +199,21 @@ Optional ideas to support assessment.
+* Short Answer Questions &mdash; Students are directed to a web page where there are several tables. Then they are asked to use an accessibility evaluation extension to find all table header and data cells and to compare these with the visual structure of the tables. Assess how students analyze if a table is coded appropriately to reflect its structure.
 * Portfolio &mdash; Students add tables to the website they are building. Assess how students code table headers and table cells, as well as how they provide a table caption and additional descriptions for tables.
 * Practical &mdash; Students are presented with a table with neither visual nor programmatic headers. Then they are asked to figure them out and decide if the table could be considered simple or complex. Assess how students identify table headers and distinguish between simple and complex tables.
 
 ## Teaching Resources
 
 Suggested resources to support your teaching:
 
-* [WAI Web accessibility Tutorials](https://www.w3.org/WAI/tutorials/) &mdash; Shows how to develop web content that is accessible to people with disabilities.
+* [Tables (WAI Web Accessibility Tutorials)](https://www.w3.org/WAI/tutorials/tables/) &mdash; Shows how to develop tables that are accessible to people with disabilities.
 * [How People with Disabilities Use the Web](/people-use-web/) &mdash; Provides stories of people with disabilities using the Web; describes types of disabilities and some of the barriers that people encounter using the Web; and introduces types of assistive technologies and adaptive strategies that some people use.
-* [Web accessibility Perspectives (videos)](/perspective-videos/) &mdash; Is a series of 1-minute videos that demonstrate that web accessibility is essential for people with disabilities and useful for all. They show accessibility features, how they impact people with disabilities, and how they benefit everyone in a variety of situations.
-* [HTML specification](https://html.spec.whatwg.org/multipage/) &mdash; The core markup language for the web, HTML, as well as numerous APIs like Web Sockets, Web Workers, localStorage, etc.
 * [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/) &mdash; Address accessibility of web content on desktops, laptops, tablets, and mobile devices.
-* [WAI ARIA](https://www.w3.org/WAI/standards-guidelines/aria/) &mdash; Provides an ontology of roles, states, and properties that define accessible user interface elements and can be used to improve the accessibility and interoperability of web content and applications.
+* [WAI-ARIA](https://www.w3.org/WAI/standards-guidelines/aria/) &mdash; Provides an ontology of roles, states, and properties that define accessible user interface elements and can be used to improve the accessibility and interoperability of web content and applications.
```

Comment:
While I agree 'ontology' is the right word, I don't think it is as clear as something like:

> "Describes the roles, states, and properties that define accessible user ..."

Reply:
Fair enough, will change that through the teaching resources section.